app/controllers/reputation_controller.rb

```diff
@@ -3,7 +3,11 @@ def notice
     if user_signed_in?
       @notice = Notice.find(params[:id])
       @evaluation_value = params[:up_down] == 'down' ? 0 : 1
-      @notice.add_or_update_evaluation(:likes, @evaluation_value, current_user)
+      if params[:up_down] == 'down'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.